stories/utils.tsx

```diff
@@ -10,3 +11,10 @@ import { StoryFnReactReturnType } from '@storybook/react/dist/client/preview/typ
 export const modalStoryDecoratorForChromatic = (
   storyFn: () => StoryFnReactReturnType,
 ) => <div style={{ width: '1200px', height: '1000px' }}>{storyFn()}</div>;
+
+/**
+ * We exclude stories that exists only for Chromatic visual testing from non-Chromatic environments
+ */
+export const excludeStoriesForNonChromaticEnvironments = (
+  stories: string[] = [],
+) => (isChromatic() ? stories : /.*/);
```

Comment:
We do this to `prevent the UI from becoming overwhelming for non-technical stakeholders` as mentioned in the PR. It seems simplest to rely on the toggle for folks to switch between the different styles rather than duplicate the stories in the sidebar, which is what happens if you remove the regex:

<img width="208" alt="Screen Shot 2021-02-16 at 10 47 21 PM" src="https://user-images.githubusercontent.com/13544620/108157783-2ce63380-70a9-11eb-9865-d130dc26659d.png">